plugin/src/main/java/io/snyk/languageserver/protocolextension/messageObjects/PublishDiagnostics316Param.java

```diff
@@ -0,0 +1,55 @@
+package io.snyk.languageserver.protocolextension.messageObjects;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class PublishDiagnostics316Param {
```

Comment:
fyi: this was added to mitigate performance issues when deserializing for every issue